pkg/proxy/iptables/proxier.go

```diff
@@ -1143,6 +1143,7 @@ func (proxier *Proxier) syncProxyRules() {
 			// If the "external" IP happens to be an IP that is local to this
 			// machine, hold the local port open so no other process can open it
 			// (because the socket might open but it would never work).
+			net.ParseIP("foobar") // force golangci-lint to fail
```

Comment:
we should force it in a staging repo too

Reply:
nevermind, I commented before I reach the last commit

Reply:
I did - look at staging/src/k8s.io/apiserver/pkg/server/handler.go